wazo_confgend/plugins/tests/test_pjsip_conf.py

```diff
@@ -142,14 +144,14 @@ def test_generate_users(self):
                     label_2=label_2,
                     name_1=name_1,
                     name_2=name_2,
-                )
+                ),
             )
 
     def test_generate_meeting_guests(self):
         output = StringIO()
-        name_1 = 'abcdef'
+        name_1 = 'abcdeféèìïöçàäỳÿüùúóíá'
         label_1 = 'meeting_guest_1_uuid'
-        name_2 = 'ghijkl'
+        name_2 = 'ghijkléèìïöçàäỳÿüùúóíá'
```

Comment:
Feel free to add some characters in utf8, but beyond standard latin1 too. Like other writing systems like kana and kanji. This is probably enough though, just a thought.

Reply:
Good point, this was pretty lazy haha.  Should perhaps add a separate test with a good variety of symbols.
However, I'm not sure of the value of this in unit tests. Everything in python 3 is utf-8 by default, so there's no expectation of any breakage in the code base itself. If there is breakage, it would be at boundaries(e.g. if encoding is changed in twisted protocol). 
Might move that kind of test there.

Reply:
No, it's a good test. Especially with writing files. Even if everything is in utf-8 by default you are reading and writing from different files and Twisted is very unclear what should be bytes vs strings and I had quite a few issues with it in provd. Most network requests are still all in bytes so you can have some niche situations where things need to be decoded and recoded.

Reply:
I added some tests in the twisted protocol tests(`wazo_confgend.tests.test_confgen`).